algorithm/classify.py

```diff
@@ -51,3 +58,64 @@ def get_redundant_data(self) -> dict[str:int]:
         Getter fn for redundant data
         """
         return self.redundant_data
+    
+    def format_name(self, name_str):
```

Comment:
Might be a good idea to separate it into another file called `utils.py` ? Pero since sabi ni cid na di naman na need i-format I guess hindi na need to?

Reply:
no, sorry for the misunderstanding. This format_name() is fine. Lets format it anyway to explicitly show that we did process the raw data and can manipulate values before writing to file

Reply:
Added that function on the utils folder!